src/services/requestServices.js

```diff
@@ -0,0 +1,20 @@
+import { Op } from 'sequelize';
+import models from '../models';
+
+const { Request } = models;
+
+/**
+   * @param {object} options query option
+   * @param {object} Request model /table
+   * @returns {Promise} Promise resolved or rejected
+   * @description get all rows where option matches
+   */
+const findRequest = (options) => {
+  Object.keys(options).forEach((k) => options[k] === undefined && delete options[k]);
+  return Request.findAll({
+    where:
+            { [Op.or]: Object.entries(options).map(([k, v]) => ({ [k]: v })) }
+  });
+};
```

Comment:
Super work bro, would be nice to have pagination too,. Like `limit` and `offset`

Reply:
yes, working on that